.github/workflows/bare-diffs.yml

```
@@ -0,0 +1,53 @@
+name: Check Tempalte Bare Minimum Diffs
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches: [main]
+    paths:
+      - 'templates/expo-template-bare-minimum/**'
+      - '.github/workflows/bare-diffs.yml'
+  pull_request:
+    paths:
+      - 'templates/expo-template-bare-minimum/**'
+      - '.github/workflows/bare-diffs.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  bare-diffs:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: 👀 Checkout
+        uses: actions/checkout@v3
+      - name: ⬢ Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: ♻️ Restore caches
+        uses: ./.github/actions/expo-caches
+        id: expo-caches
+        with:
+          yarn-docs: 'true'
```

```suggestion
          yarn-docs: 'true'
          yarn-tools: 'true'
```

since we will launch expotools